### PR TITLE
[FIS-280 ] Feature - mobile payment link component created in UI

### PR DESCRIPTION
**Pull request by IgorZ07**
Screen Shot 2019-09-06 at 2 16 01 PM

### CODE REVIEW

File: src/components/mobile-payment-link/MobilePaymentLink.vue [hunk 1]
```diff
@@ -0,0 +1,53 @@
+<template>
+  <div class="mobile-payment-link">
+    <div class="header">
```
Comment (WouterSioen) [inline src/components/mobile-payment-link/MobilePaymentLink.vue]: Isn't `div class="header"` a bit of an anti-pattern? Shouldn't the `header` html tag be used to be semantically correct?

File: src/components/mobile-payment-link/MobilePaymentLink.vue [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      <span class="headerText">Payment link</span>
+      <span class="copyButton" @click.stop.prevent="copyLink">Copy</span>
```
Comment (WouterSioen) [inline src/components/mobile-payment-link/MobilePaymentLink.vue]: TIL: didn't know a `.stop.prevent` thing existed, very cool 👌

File: src/components/mobile-payment-link/MobilePaymentLink.vue [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    link: { type: String, default: "" }
+  },
+  methods: {
+    copyLink() {
+      let textToCopy = document.querySelector("#linkToCopy");
+      textToCopy.setAttribute("type", "text");
+      textToCopy.select();
+      document.execCommand("copy");
+      textToCopy.setAttribute("type", "hidden");
+      window.getSelection().removeAllRanges();
```
Comment (WouterSioen) [inline src/components/mobile-payment-link/MobilePaymentLink.vue]: Is this working in all browsers we want to support? Afaik, copy pasting had some cross-browser problems until recently?

File: src/components/mobile-payment-link/MobilePaymentLink.vue [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+};
+</script>
+
+<style lang="scss" scoped>
+@import "../../assets/style/main";
+
+.mobile-payment-link {
+  display: flex;
+  flex-direction: column;
+  font-family: "Montserrat", sans-serif !important;
```
Comment (WouterSioen) [inline src/components/mobile-payment-link/MobilePaymentLink.vue]: Shouldn't this be a css variable?

File: stories/index.js [hunk 5]
```diff
@@ -211,4 +211,10 @@ storiesOf("Marketing Thumb", module)
       text="hundreds of fishing experiences around the world handpicked by Fishtripr."
       />`
   );
+storiesOf("Mobile Payment Link", module)
+  .add("Basic", () =>
+    `<mobile-payment-link
+      link="https://www.imdb.com/title/tt0068646/?ref_=ttls_li_tt"
```
Comment (WouterSioen) [inline stories/index.js]: I prefer to always use example.com links in non-production data. That domain is created to be used as a dummy placeholder. If you don't do this, you for example easily end up also using email addresses that can exist for real.

File: src/components/mobile-payment-link/MobilePaymentLink.vue (hunk 3 above)
Comment (IgorZ07) [reply src/components/mobile-payment-link/MobilePaymentLink.vue]: Here is compatibility as of today, shall I change the code for any browser?

Screen Shot 2019-09-09 at 12 44 35 PM

File: src/components/mobile-payment-link/MobilePaymentLink.vue (hunk 2 above)
Comment (KevinTss) [reply src/components/mobile-payment-link/MobilePaymentLink.vue]: https://vuejs.org/v2/guide/events.html#Event-Modifiers

File: src/components/mobile-payment-link/MobilePaymentLink.vue (hunk 3 above)
Comment (WouterSioen) [reply src/components/mobile-payment-link/MobilePaymentLink.vue]: That seems better than I remembered. Should be fine indeed

**Pull request merged**